test/api/cluster.go

```diff
@@ -0,0 +1,26 @@
+package api
```

Comment:
Same rename like `testdata` as discussed above?

Reply:
oh,i forget this,maybe this dir is better?
```
--test
   |--testdata
       |-- api
            |--clister.go
            |--k8s.go 
```
because testdata looks like the data of test,like yaml .Bug k8s.go is the base function of test

Reply:
Sounds OK.

Reply:
done